ecosystem/typescript/sdk_v2/tests/e2e/api/general.test.ts

```diff
@@ -20,7 +20,7 @@ describe("general api", () => {
   test("it fetches block data by block height", async () => {
     const config = new AptosConfig({ network: Network.LOCAL });
     const aptos = new Aptos(config);
-    const blockHeight = 100;
+    const blockHeight = 1;
```

Comment:
is this supposed to be here?  just checking because I don't have enough context to understand changing 100 -> 1

Reply:
yes it is just a test for the get block height query. block height = 100 might be to high for example when you start a local node, block height hasn't reach that number yet. so changing to 1. can read more here https://aptos.dev/reference/glossary/#version